src/main/web_src/src/pages/gruppe/PersonListe/PersonListe.js

```diff
@@ -26,29 +25,38 @@ const ikonTypeMap = {
 export default function PersonListe({
 	isFetching,
 	personListe,
-	searchActive,
+	gruppeInfo,
+	identer,
+	sidetall,
+	setSidetall,
+	sideStoerrelse,
+	setSideStoerrelse,
 	visPerson,
 	iLaastGruppe,
 	fetchTpsfPersoner
 }) {
-	useMount(fetchTpsfPersoner)
-
 	const [isKommentarModalOpen, openKommentarModal, closeKommentarModal] = useBoolean(false)
 	const [selectedIdent, setSelectedIdent] = useState(null)
 
+	useEffect(() => {
+		fetchTpsfPersoner()
+	}, [identer])
+
 	if (isFetching) return <Loading label="laster personer" panel />
 
+	if (visPerson && personListe && window.sessionStorage.getItem('sidetall')) {
```

Comment:
Ser ikke ut til at denne fungerer helt som den skal 🤔  For jeg får ikke til å søke på person - denne blir undefined, og dermed blir sidetall NaN. Så da får jeg ikke navigert til personer som ikke ligger på første side. Mulig å finne ut av? 😁